examples/az.json

```diff
@@ -9,7 +9,6 @@
                 "useCloudControllerManager": true,
                 "customCcmImage": "gcrio.azureedge.net/google_containers/cloud-controller-manager-amd64:v1.15.0",
                 "customHyperkubeImage": "gcrio.azureedge.net/google_containers/hyperkube-amd64:v1.15.0",
-                "networkPolicy": "none",
```

Comment:
Could you explain why the "networkPolicy" filed is removed here?

Reply:
I'm following https://github.com/bowen5/aks-engine/blob/master/docs/topics/clusterdefinitions.md and seems "none" is not a valid value for "networkPolicy" property for now. Or shall I change this value to "azure" or something else? Thank you.

Reply:
OK, then let's remove it.